Sources/XcodeProj/Project/XcodeProj.swift

```diff
@@ -8,6 +8,9 @@ public final class XcodeProj: Equatable {
     /// Project workspace
     public var workspace: XCWorkspace
 
+    /// The  path to the `.xcodeproj` directory.
+    private let projectPath: Path?
```

Comment:
Do we want to make it public, so it can be accessed outside of this module?

Reply:
🤦 Yes that was the whole point, earth to Andy lol

Reply:
haha, could happen to anyone 😂